diff --git a/node_modules/@types/progress/index.d.ts b/node_modules/@types/progress/index.d.ts
index c1e199e..89ce598 100644
--- a/node_modules/@types/progress/index.d.ts
+++ b/node_modules/@types/progress/index.d.ts
@@ -70,7 +70,7 @@ declare class ProgressBar {
     /**
      * "interrupt" the progress bar and write a message above it.
      */
-    interrupt(message: string): void;
+    interrupt(message: string, draw?:boolean): void;
 
     /**
      * Terminates a progress bar.
